### ISSUE TITLE
buildroot-eln.yaml keeps getting reorganized every hour

### ISSUE THREAD

**Issue by hroncok** [minimization/content-resolver-input#420]
I've noticed that most of the changes in https://github.com/minimization/content-resolver-input/commits/master/configs/buildroot-eln.yaml are just changing the order of packages. May the packages please be sorted, so the changes are easier to review? Thanks.

**Comment by jwboyer** [comment]
@tdawson is this something you can look at?

**Comment by tdawson** [comment]
Ya, that thing is massive and it changes alot, even without sorting changes.
I thought I was sorting things, but maybe I missed something, so yes, I will look into this.

**Comment by tdawson** [comment]
I was missing some 'sorts'.  I have now triple checked it, and it hopefully has all packages in sorted order now, in all parts of the yaml file.  I believe this is fixed, but will leave this open and check tomorrow to make sure it lasts and there wasn't some other missing sort.

**Comment by tdawson** [comment]
Looking at the git log, although there are still changes about every two hours, it looks like they are real changes.  First there were alot of ghc changes and then a bunch of rust changes.  But I did not see any "changed order" changes.

Let me know if you think I've fixed the original problem.

I'm going to file a separate issue because I don't think the buildroot files should be in this same repo.  It causes too much churn for people to work on their workload yaml files.
